### PR TITLE
Added important note #3077

### PR DESCRIPTION
**Pull request by j0rt3g4**
I've tried to reproduce the issue that the person is reporting but I was unable to do so.
Added an important note related to E4 Licenses.

Copy review @nenonix @JohanFreelancer9 is required on this one guys.
Thank you in advance!

### CODE REVIEW

File: Skype/SfbOnline/skype-for-business-and-microsoft-teams-add-on-licensing/license-options-based-on-your-plan/office-365-enterprise-e1-e3-e4.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ If you have already purchased the [Office 365 Enterprise E1](https://products.of
 |**Cloud Voicemail** <br/> |For E1 and E4 plans, buy Exchange Online Plan 2. <br/>Or, switch to an Office Suite that comes with Exchange Online Plan 2, such as Office 365 E3. |
 |**Calling Plans** <br/> |1. **Phone System** add-on <br/> 2. [Calling Plan](/MicrosoftTeams/calling-plans-for-office-365): you can only buy this after you buy the **Phone System** add-on. <br/> |
    
+ > [!IMPORTANT]
+ > Office 365 E4 licenses has been officially retired, please view: https://support.office.com/article/important-information-for-office-365-enterprise-e4-customers-f9572348-43a2-43fa-a3d8-3b6c9c042147
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/skype-for-business-and-microsoft-teams-add-on-licensing/license-options-based-on-your-plan/office-365-enterprise-e1-e3-e4.md]:
```suggestion
 > Office 365 E4 licenses have been officially retired. Please see [Important information for Office 365 Enterprise E4 customers](https://support.office.com/article/important-information-for-office-365-enterprise-e4-customers-f9572348-43a2-43fa-a3d8-3b6c9c042147) for more details.
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by rjagiewich** [comment]
j0rt3g4 -  Merged this PR because the addition was straightforward. It also appears that the copyedit was merged.

**Comment by j0rt3g4** [comment]
Thank you @rjagiewich  for taking the taking to explain it.
Either way the important part is to get the docs better! :) thank you!

**Pull request merged**